pkgs/development/python-modules/dash-core-components/default.nix

```
@@ -0,0 +1,23 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+}:
+
+buildPythonPackage rec {
+  pname = "dash-core-components";
+  version = "1.7.0";
+
+  src = fetchurl {
```

There's already a helper for this, see https://github.com/NixOS/nixpkgs/blob/211ed174657712a00fd3bcabd590d2e7b7a25e10/pkgs/development/python-modules/requests/default.nix for an example
```suggestion
  src = fetchPypi {
```

---

@jonringer I initially gave up on `fetchPypi` because it wasn't working for me. I just realized it's because the package name should have underscores instead of dashes. It's now fixed for all packages in 4f0952022dcd5314aa6b56a98901155110c3294a.